index.js

```diff
@@ -35,6 +35,9 @@ module.exports = function (opts) {
 
 			file.contents = new Buffer(res.code);
 			file.path = replaceExt(file.path, '.js');
+			file.metadata = {
```

Comment:
`file.metadata` => `file.babel` so not to potentially conflict with other plugins.